### PR TITLE
FRAAS-3873 Simplify forge install

### PR DESCRIPTION
**Pull request by livstowe**
[JIRA](https://bugster.forgerock.org/jira/browse/FRAAS-3873)

Simplifies the forge install script.
Makes setting optional vars consistent.
Remove the need for the AQL query when finding the latest version.

### CODE REVIEW

File: install_forge.sh [hunk 1]
```diff
@@ -7,19 +7,16 @@
...
 # Optional Vars (the defaults will work for most):
 # • ARTIFACTORY_API_KEY : API key for Artifactory auth
-# • FORGE_VERSION       : Version to install (i.e, 2.1.0, 2.1.*, 2.*, *)
+# • FORGE_VERSION       : Version to install
 # • FRAAS_CONFIG_PATH   : Path to the FRaaS config file
 # • INSTALL_PATH        : Path to save Forge to
-#
-# Pass "-y" or "--yes" to skip user prompts:
-#   bash forge_intall.sh -y
+# • VERIFY_INSTALL      : "yes" to verify before install, "no" to skip
```
Comment (livstowe) [inline install_forge.sh:13]: Makes the `VERIFY_INSTALL` consistent with other optional vars.

File: install_forge.sh [hunk 2]
```diff
@@ -119,40 +77,40 @@ EOF
   exit 1
 fi
 
-info "Searching for the Forge binary..."
-
-QUERY_VERSION="forge-${FORGE_VERSION}-${OS}"
```
Comment (livstowe) [inline install_forge.sh:124]: The AQL query is replaced with a check of the `latest` file from Artifactory.

File: install_forge.sh [hunk 3]
```diff
@@ -119,40 +77,40 @@ EOF
...
-  -H "Content-Type: text/plain" \
-  -d "${AQL_QUERY}")
+# Grab the latest version if not specified
+if [[ -z "${FORGE_VERSION}" ]]; then
+  echo "FORGE_VERSION was not specified, finding the latest version"
+  FORGE_VERSION=$(curl -fsSL -H "${AUTH}" "${REPO_URL}/latest")
+fi
 
-# Using sed to avoid requiring a json parser to install Forge
-FORGE_BINARY=$(echo "${QUERY_RESULT}" | sed -n 's/.*"name" : "\(.*\)",/\1/p')
```
Comment (livstowe) [inline install_forge.sh:142]: No need to parse a response from Artifactory anymore. 🎉

**Review by swensone — APPROVED** [review]
Some nice changes and simplifications here.

**Pull request merged**